WalletWasabi/WabiSabi/Backend/Rounds/RoundParameters.cs

```diff
@@ -120,10 +125,25 @@ public Transaction CreateTransaction()
 	/// <remarks>It won't be smaller than min allowed output amount.</remarks>
 	public Money CalculateMinReasonableOutputAmount()
 	{
-		var minEconomicalOutput = MiningFeeRate.GetFee(
-						Math.Max(
-							ScriptType.P2WPKH.EstimateInputVsize() + ScriptType.P2WPKH.EstimateOutputVsize(),
-							ScriptType.Taproot.EstimateInputVsize() + ScriptType.Taproot.EstimateOutputVsize()));
+		var minEconomicalOutput = MiningFeeRate.GetFee(MaxVsizeInputOutputPair);
 		return Math.Max(minEconomicalOutput, AllowedOutputAmounts.Min);
 	}
+
+	/// <returns>Smallest effective denom that's larger than min reasonable output amount. </returns>
+	public Money CalculateSmallestReasonableEffectiveDenomination()
```

Comment:
This function slightly breaks the API of WabiSabiClientLibrary. We do not send `RoundParameters` to the library since it contains a lot of useless fields. Which means we are not able to call `CalculateSmallestReasonableEffectiveDenomination` without initializing a dummy `RoundParameters`. Would you mind adding a static variant of `CalculateSmallestReasonableEffectiveDenomination`?

Reply:
Yes, this is exactly what I meant.

Reply:
Done: https://github.com/zkSNACKs/WalletWasabi/pull/10842/commits/11470dc90316b7145063a2f0786fa8c612a38361

Will it suffice or would you also like a static `CalculateMinReasonableOutputAmount`?